config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -103,6 +103,44 @@ periodics:
     testgrid-alert-email: release-team@kubernetes.io
     description: 'Ends up running: make quick-release'
 
+- interval: 5m
+  name: ci-kubernetes-build-fast-canary
+  cluster: k8s-infra-prow-build
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/bootstrap:v20200924-7fcf543
+      args:
+      - --repo=k8s.io/kubernetes
+      - --repo=k8s.io/release
+      - --root=/go/src
+      - --timeout=40
+      - --scenario=kubernetes_build
+      - --
+      - --allow-dup
+      - --fast
+      - --release=k8s-release-dev/ci-canary
```

Comment:
I'll merge this as-is just to see if the job runs successfully, but for followup this should be different bucket / same path.  I think what you have here is different bucket / different path

Reply:
ok, followup PR: https://github.com/kubernetes/test-infra/pull/19518